mlflow/pytorch/__init__.py

```diff
@@ -285,6 +287,92 @@ def gen_data():
     )
 
 
+@experimental
+def log_state_dict(state_dict, artifact_path, **kwargs):
+    """
+    Log a state dict as an MLflow artifact for the current run.
+
+    :param state_dict: PyTorch model state dict to be saved.
```

Comment:
```suggestion
    :param state_dict: PyTorch state dict to be saved.
```

Can we remove `model` here?

Reply:
done